mock/py/mockbuild/package_manager.py

```diff
@@ -328,11 +328,34 @@ def copy_certs(self):
                 # when mock is not run on Fedora or EL
                 self.buildroot.root_log.debug("ca bundle not found on host")
 
+    @traceLog()
+    def copy_extra_certs(self):
+        extra_certs = self.config['ssl_extra_certs']
+        if extra_certs:
+            self.buildroot.root_log.debug('copying extra certificates into chroot')
+            for cert_src, cert_dest in zip(extra_certs[::2], extra_certs[1::2]):
+                host_cert_src = os.path.realpath(cert_src)
+                chroot_cert_dest = self.buildroot.make_chroot_path(cert_dest)
+                chroot_cert_dir = os.path.dirname(chroot_cert_dest)
+                try:
+                    os.makedirs(chroot_cert_dir)
+                except FileExistsError:
+                    pass  # for repeated attempts
+
+                try:
+                    shutil.copy(host_cert_src, chroot_cert_dest)
+                except FileNotFoundError:
+                    # when mock is not run on Fedora or EL
+                    self.buildroot.root_log.debug("extra certificates not found on host")
+
+
     def initialize(self):
         self.copy_gpg_keys()
         self.copy_certs()
+        self.copy_extra_certs()
```

Comment:
Can this call be dropped?  The later call seems better, as that is only affecting the bootstrap chroot... and we don't really want to install the chroots to the target chroot?

Reply:
I tested it and yes it can be dropped. Done in https://github.com/rpm-software-management/mock/pull/801/commits/c69fda9051c491bf91c22403f1ba4e301e2c2fad